pghrep/templates/A005.tpl

```diff
@@ -1,31 +1,26 @@
-# Extensions #
+# {{ .checkId }} Extensions #
 
-## Current values ##
+## Observations ##
 
-### Master DB server is `{{.hosts.master}}` ###
-Master DB server is {{.hosts.master}}
-{{ range $key, $value := (index (index .results .hosts.master) "data") }}
-Extension: `{{ $key }}`, Installed version: `{{ $value.installed_version}}`
+### Master (`{{.hosts.master}}`) ###
+Extension name | Installed version
```

Comment:
what about available version and the flag showing if the installed version is up-to-date?